lib/internal/modules/esm/resolve.js

```diff
@@ -1026,33 +1026,9 @@ function throwIfInvalidParentURL(parentURL) {
  * @param {string} [context.parentURL] - The URL of the parent module.
  * @param {string[]} [context.conditions] - The conditions for resolving the specifier.
  */
-function defaultResolve(specifier, context = {}) {
+function defaultResolve(specifier, context = {}, importMap) {
```

Comment:
We can’t change this signature without breaking the customization hooks. Just make `importMap` a new property on `context`.

Reply:
Ah, saw this after my comment on esm/loader. +1 🙂